DependencyInjection/HttplugExtension.php

```diff
@@ -27,6 +27,13 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.xml');
         $loader->load('plugins.xml');
         $loader->load('discovery.xml');
+        if ($config['toolbar']['enabled'] && $container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
```

Comment:
> or is debug toolbar hard coupled to kernel.debug?

No it is not. To get the toolbar you need `framework.profiler` enabled and of course the WebProfilerBundle. There is nothing that ties it to kernel.debug. So this is technically wrong, but it works for most cases. Others are doing the same.

https://github.com/egeloen/IvoryHttpAdapterBundle/blob/master/DependencyInjection/IvoryHttpAdapterExtension.php#L62

I could use a compiler pass and check for the `web_profiler.debug_toolbar.mode` parameter if it is enabled or disabled. But I think that will complicate how I inject the HistoryPlugin because that would also be done in the compiler pass. 

Question: Are we happy with postponing support for the use case where you want the debug Httplug request via the toolbar with `kernel.debug: false`?